protocols/raft/src/main/java/io/atomix/protocols/raft/storage/snapshot/SnapshotDescriptor.java

```diff
@@ -30,7 +30,23 @@
  */
 public final class SnapshotDescriptor implements AutoCloseable {
   public static final int BYTES = 64;
-  public static final int VERSION = 1;
+
+  // Current snapshot version.
+  private static final int VERSION = 1;
```

Comment:
one observation here is that version was never incremented and there is no backwards compatibility with version one, are snapshots always storage level memory? maybe it's not an actual issue.

Reply:
@deepthidevaki how has this been performing in production, did you see this comment?

Reply:
Sorry. Yes. Compatibility could be an issue. We never ran into that because we never had to upgrade from previous version.